spec/classes/katello_application_spec.rb

```diff
@@ -9,7 +9,11 @@
         {
           :package_names          => ['tfm-rubygem-katello'],
           :enable_ostree          => false,
-          :rubygem_katello_ostree => 'tfm-rubygem-katello_ostree',
+          :enable_yum             => true,
+          :enable_file            => true,
+	  :enable_puppet          => true,
```

Comment:
This passed linting ? :thinking:

Reply:
Its because you used tabs, you cowboy :cowboy_hat_face:

Reply:
errr what???  /me checks vim configuration

Reply:
:cry: newer-ish box didn't have my normal config

Reply:
We apologies for the tabs, those responsible have now been sacked.